scipy/stats/tests/test_distributions.py

```diff
@@ -9646,6 +9646,53 @@ def test_sf(self):
         assert_allclose(sf1, sf0)
 
 
+def test_irwinhall():
+    unif = stats.uniform(0, 1)
+    ih1 = stats.irwinhall(1)
+
+    pts = np.linspace(0, 1, 100)
+    pdf_unif = unif.pdf(pts)
+    pdf_ih1 = ih1.pdf(pts)
+    assert_allclose(pdf_ih1, pdf_unif) # should be identical to uniform
+
+    cdf_unif = unif.cdf(pts)
+    cdf_ih1 = ih1.cdf(pts)
+    assert_allclose(cdf_ih1, cdf_unif) # should be identical to uniform
+
+    ih2 = stats.irwinhall(2)
+    pdf_ih2 = ih2.pdf(1)
+    assert_allclose(pdf_ih2, 1) # should be 1 at x=1
+
+    npts = 100
+    pts = np.linspace(0, 2, npts)
+    expected = np.linspace(0, 2, npts)
+    expected[npts//2:] = 2 - expected[npts//2:]
```

Comment:
```suggestion
    npts = 101
    pts = np.linspace(0, 2, npts)
    expected = np.linspace(0, 2, npts)
    expected[(npts+1)//2:] = 2 - expected[(npts+1)//2:]
```
If you want to test the midpoint, I think this is right.